src/public/legal.md

```diff
@@ -0,0 +1,131 @@
+<!--
+SPDX-FileCopyrightText: 2024 PNED G.I.E.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Website Terms
+
+## Website owner
+
+This website is owned and operated by GDI Coordination, 6 avenue des Hauts-Fourneaux, L-4362 Esch-sur-Alzette, Grand-Duchy of Luxembourg. GDI Coordination can be contacted at gdi-coordination@elixir-europe.org.
+
+## Conditions of use
+
+By using this website, users agree to abide by these website terms (in their most recent version, as available on the website). GDI Coordination reserves the right to modify this website and the website terms from time to time.
+
+Users acknowledge having verified that their equipment does not contain any viruses or any other malicious software. Users are solely responsible for the selection, use, and interpretation of the data, information, and documents available on this website.
+
+## LIABILITY
```

Comment:
Is this normal that some headings are All Caps but some are Title case?

Reply:
its written the same way as the legal page in lnds website, which was the target of this ticket, I also thought it looked weird, if you want me to change them let me know

Reply:
ah! if it is case, let's ask before changing, so good 👍